packages/block/test/testdata/testdata.json

```diff
@@ -25,7 +25,7 @@
           "data": "0x",
           "gasLimit": "0x55f0",
           "gasPrice": "0x0a",
-          "nonce": "0x00",
+          "nonce": "0x",
```

Comment:
would it be better to use `0x0` to represent the number zero? i would think `0x` represents null. maybe update a few of them so we have both cases covered?